backend/app/views/spree/admin/shared/_menu.html.erb

```diff
@@ -1,9 +1,5 @@
-<nav id="admin-menu" data-hook>
-  <div class="container">
-    <div class="sixteen columns main-menu-wrapper">
-      <ul data-hook="admin_tabs" class="fullwidth-menu">
-        <%= render :partial => 'spree/admin/shared/tabs' %>
-      </ul>
-    </div>
-  </div>
+<nav class="admin-menu">
+  <ul data-hook="admin_tabs">
+    <%= render :partial => 'spree/admin/shared/tabs' %>
```

Comment:
Since we're touching all of these files anyways I'd love for us to switch to the new hash syntax.